customer-commons/src/main/java/no/unit/nva/customer/model/CustomerMapper.java

```diff
@@ -0,0 +1,82 @@
+package no.unit.nva.customer.model;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.net.URI;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import nva.commons.utils.JsonUtils;
+
+public class CustomerMapper {
+
+    public static final URI NO_CONTEXT = null;
+    public static URI context;
+
+    static {
+        context = URI.create("https://bibsysdev.github.io/src/customer-context.json");
+    }
+
+    private final ObjectMapper objectMapper;
+    private final String namespace;
+
+    public CustomerMapper(String namespace) {
+        this.namespace = namespace;
+        objectMapper = JsonUtils.objectMapper;
+    }
+
+    /**
+     * Map from Customer from Db to Dto version.
+     *
+     * @param customerDb    customerDb
+     * @return  customerDto
+     */
+    public CustomerDto toCustomerDto(CustomerDb customerDb) {
+        CustomerDto customerDto = objectMapper.convertValue(customerDb, CustomerDto.class);
+        URI id = toId(customerDb.getIdentifier());
+        customerDto.setId(id);
+        customerDto.setContext(context);
+        return customerDto;
+    }
+
+    /**
+     * Map from Customer from Db to Dto version without context object.
+     *
+     * @param customerDb    customerDb
+     * @return  customerDto
+     */
+    public CustomerDto toCustomerDtoWithoutContext(CustomerDb customerDb) {
+        CustomerDto customerDto = toCustomerDto(customerDb);
+        customerDto.setContext(NO_CONTEXT);
+        return customerDto;
+    }
+
+    /**
+     * Map from list of Customers from Db to Dto version.
+     *
+     * @param customersDbs  list of CustomerDb
+     * @return  customerList
+     */
+    public CustomerList toCustomerList(List<CustomerDb> customersDbs) {
+        List<CustomerDto> customerDtos = customersDbs.stream()
+            .map(this::toCustomerDtoWithoutContext)
+            .collect(Collectors.toList()
+            );
+        return CustomerList.of(customerDtos);
```

Comment:
https://github.com/BIBSYSDEV/nva-customer-api/pull/25#discussion_r492930458

@axthosarouris : It is a `CustomerList` here in the return.